testgui/mac_support_cocoa.m

```diff
@@ -8,10 +8,15 @@
 #include <fx.h>
 #import <Cocoa/Cocoa.h>
 
+// macOS 10.12 deprecated NSAnyEventMask in favor of NSEventMaskAny
+#if MAC_OS_X_VERSION_MAX_ALLOWED < 101200
```

Comment:
lets more friendly name for this constant, e.g.: MAC_OS_X_VERSION_10_12

Reply:
This constant isn't available under macOS 10.12 so we'd have to add
```c
#ifndef MAC_OS_X_VERSION_10_12
#define MAC_OS_X_VERSION_10_12 101200
#endif
```
Should I do that?

Reply:
what about [this solution](https://stackoverflow.com/a/41497822/3697939) ?
your suggestion is fine by me as well
anything is better than magic numbers